lib/directory-colorfy.js

```diff
@@ -37,49 +37,51 @@
 		this.colors = this.options.colors || {};
 	};
 
-	DirectoryColorfy.prototype.convert = function(){
-		var promises = this.files.map(function(self, file){
-			return new Colorfy( file, self.colors, {
-				dynamicColorOnly: self.options.dynamicColorOnly
-			});
-		}.bind( null, this ));
+	DirectoryColorfy.prototype.convert = function() {
+        var promises = this.files.map(function(self, file) {
+            return new Colorfy(file, self.colors, {
+                dynamicColorOnly: self.options.dynamicColorOnly
+            });
+        }.bind(null, this));
 
-		return new RSVP.Promise(function(self, colorfied, resolve, reject){
-			var filesWritten = [];
-			var converted = [];
+        return new RSVP.Promise(function(self, colorfied, resolve, reject) {
+            var filesWritten = [];
+            var converted = [];
 
-			var handlers = {
-				collect: function( dc, err, output ){
-				if( err ){
-					reject( err );
-				} else {
-					converted.push( output );
-					if( converted.length === colorfied.length ){
-						filesWritten = converted.map(function( col ){
-							var c = new ColorfyFromOptions( col );
-							return c.writeFiles( this.output );
-						}, dc )
-						.reduce(function(a, b){
-							return a.concat( b );
-						});
-					}
-				}
-				}.bind( null, self ),
+            var handlers = {
+                collect: function(dc, err, output) {
+                    if (err) {
+                        reject(err);
+                    } else {
+                        converted.push(output);
+                        if (converted.length === colorfied.length) {
+                            filesWritten = converted.map(function(col) {
+                                    var c = new ColorfyFromOptions(col);
+                                    return c.writeFiles(this.output);
+                                }, dc)
+                                .reduce(function(a, b) {
+                                    return a.concat(b);
+                                });
+                        }
+                    }
+                }.bind(null, self),
 
-				complete: function() {
-					resolve(filesWritten);
-				}
-			};
+                complete: function() {
+                    if (converted.length === colorfied.length) {
+                        resolve(filesWritten);
+                    }
+                }
+            };
 
-			if( colorfied.length === 0 ){
-				resolve( filesWritten );
-			} else {
-				colorfied.forEach(function( colorfy ){
-					dirConvert( colorfy, handlers );
-				});
-			}
-		}.bind( null, this, promises ));
-	};
+            if (colorfied.length === 0) {
+                resolve(filesWritten);
+            } else {
+                colorfied.forEach(function(colorfy) {
+                    dirConvert(colorfy, handlers);
+                });
+            }
+        }.bind(null, this, promises));
+    };
```

Comment:
Just checking in, these just seem to be whitespace changes so I will ignore them unless otherwise necessary.